src/Entity/Job.php

```diff
@@ -133,6 +135,9 @@ class Job
     #[ApiFilter(filterClass: SearchFilter::class, strategy: SearchFilterInterface::STRATEGY_PARTIAL)]
     private ?LocationType $locationType = null;
 
+    #[ORM\ManyToMany(targetEntity: Tags::class, mappedBy: 'job')]
+    private Collection $tag;
```

Comment:
`$tags` maybe?

Reply:
Yes but there can't remove/change the past fielsName tag without delete data.